packages/web/src/pages/earn/pool/[pool-path]/[position-id]/reposition.tsx

```diff
@@ -0,0 +1,53 @@
+import Footer from "@components/common/footer/Footer";
+import BreadcrumbsContainer from "@containers/breadcrumbs-container/BreadcrumbsContainer";
+import HeaderContainer from "@containers/header-container/HeaderContainer";
+import RepositionContainer from "@containers/reposition-container/RepositionContainer";
+import { useLoading } from "@hooks/common/use-loading";
+import { useWindowSize } from "@hooks/common/use-window-size";
+import { useGnotToGnot } from "@hooks/token/use-gnot-wugnot";
+import RepositionLayout from "@layouts/reposition/RepositionLayout";
+import { DeviceSize } from "@styles/media";
+import { useRouter } from "next/router";
+import { useMemo } from "react";
+import { useGetPoolDetailByPath } from "src/react-query/pools";
+
+export default function EarnAdd() {
+  const { width } = useWindowSize();
+  const router = useRouter();
+  const poolPath = router.query["pool-path"] || "";
+  const { data, isLoading } = useGetPoolDetailByPath(poolPath as string, {
+    enabled: !!poolPath,
+  });
+  const { getGnotPath } = useGnotToGnot();
+  const { isLoadingCommon } = useLoading();
+
+  const listBreadcrumb = useMemo(() => {
+    return [
+      { title: "Earn", path: "/earn" },
+      {
+        title:
+          width > DeviceSize.mediumWeb
+            ? `${getGnotPath(data?.tokenA).symbol}/${
+                getGnotPath(data?.tokenB).symbol
+              } (${Number(data?.fee) / 10000}%)`
+            : "...",
+        path: `/earn/pool/${router.query["pool-path"]}`,
+      },
+      { title: "Reposition", path: "" },
+    ];
+  }, [data, width]);
+
+  return (
+    <RepositionLayout
+      header={<HeaderContainer />}
+      breadcrumbs={
+        <BreadcrumbsContainer
+          listBreadcrumb={listBreadcrumb}
+          isLoading={isLoadingCommon || isLoading}
```

Comment:
Is there any way to improve BreadcrumbsContainer ?
The data that changes every time the width changes is the data in the 2nd step, which becomes "..." in the second step. 
It seems like it could be improved if there are other types of changes.

Reply:
I will update this in the new PR. because some screen use ..., some screen have different breakpoint. is it okay.

Reply:
Great, let's do that.